docs/index.md

```diff
@@ -15,6 +15,32 @@ You can set environment variable `SECRETHUB_CREDENTIAL` or read it from disk usi
 
 Use the navigation to the left to read about the available resources.
 
+## Installation
+
+### Terraform 0.13 and later
+
+To install this provider, copy and paste this code into your Terraform configuration. Then, run `terraform init`.
+
+```hcl
+terraform {
+  required_providers {
+    secrethub = {
+      source = "secrethub/secrethub"
+      version = "1.2.2"
+    }
+  }
+}
+```
+
+### Terraform 0.12 and earlier
+
+For Terraform 0.12 and earlier, you have to download the binary for your operating system and architecture from [GitHub releases](https://github.com/secrethub/terraform-provider-secrethub/releases) yourselves and place it in `~/.terraform.d/plugins` (`%APPDATA%\terraform.d\plugins` on Windows).
+
+For linux amd64, you can do so with the following command:
+```sh
+mkdir -p ~/.terraform.d/plugins && curl -SfL https://github.com/secrethub/terraform-provider-secrethub/releases/latest/download/terraform-provider-secrethub-linux-amd64.tar.gz | tar zxf - -C ~/.terraform.d/plugins
+```
```

Comment:
We could also just link to our nice install tabs here: https://secrethub.io/docs/guides/terraform/#install
So you get macOS, Windows, and 386 too. And we keep it short here, e.g.:

````suggestion
For Terraform 0.12 and earlier, you have to [place the binary in the plugin directory](https://secrethub.io/docs/guides/terraform/#install)
````